### PR TITLE
add ssd_detect.py

### PR DESCRIPTION
**Pull request by ZhangXinNan**
添加一个ssd_detect.py文件

### CODE REVIEW

File: ssd_detect.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        for i in xrange(0, num_labels):
+            if label == labelmap.item[i].label:
+                found = True
+                labelnames.append(labelmap.item[i].display_name)
+                break
+        assert found == True
+    return labelnames
+
+class CaffeDetection:
+    '''使用caffe作检测'''
```
Comment (weiliu89) [inline ssd_detect.py]: Could you remove it or translate it to english? Thanks!

**Comment by weiliu89** [comment]
Sorry, one more thing: could you move the file to examples/ssd/? Thanks! After this, I think we could merge it.

**Comment by ZhangXinNan** [comment]
If it's OK, I would merge commits.

**Comment by weiliu89** [comment]
Thanks

**Pull request merged**